src/operation/create_indexes/mod.rs

```diff
@@ -77,19 +71,11 @@ impl Operation for CreateIndexes {
 
     fn handle_response(
         &self,
-        response: Response,
+        _response: WriteConcernOnlyBody,
```

Comment:
We'll need a call to `response.validate()` in this method somewhere to check for write concern errors.